docs/testplan/Chassis_GCU.md

```diff
@@ -0,0 +1,118 @@
+
+### Test PRs
+
+Above changes were added via Test PRs:
+- [Generalizing GCU test suite to verify both ip types (IPV4, IPV6) for BGP Neighbors](https://github.com/sonic-net/sonic-mgmt/pull/13650)
+- [GCU adding Multi-ASIC support in existing test code base](https://github.com/sonic-net/sonic-mgmt/pull/14070)
+
+## Testbed
+
+The test will run on T2 testbeds.
+
+## Setup Configuration
+
+TBD
+
+## Testing Plan
+
+### Test Case # 1 - Add Cluster With Basic QoS
+
+#### Test Objective
+
+To verify updates in config paths:
+    /asic/BGP_NEIGHBOR
+    /asic/DEVICE_NEIGHBOR
+    /asic/DEVICE_NEIGHBOR_METADATA
+    /asic/PORTCHANNEL_MEMBER
+    /asic/PORTCHANNEL_INTERFACE
+    /asic/INTERFACE
+    /asic/PORT
+    /localhost/BGP_NEIGHBOR
+    /localhost/DEVICE_NEIGHBOR_METADATA
+    /localhost/INTERFACE
+    /localhost/PORTCHANNEL_INTERFACE
+    /localhost/PORTCHANNEL_MEMBER
+
+#### Test Requirements:
+
+At least two frontend DUT hosts are required to perform traffic. Modifications via apply-patch are applied on the downstream frontend DUT host. The scenario verifies data traffic from upstream to downstream and downstream to downstream.
+
+#### Testing Steps
+
+- Select a random ASIC from the downstream host.
+- Select a random BGP neighbor fro-m that namespace and announce a static route for a DST_IP that is advertised only via this neighbor.
+- Verify the route table in the downstream DUT host to ensure that the static route is visible.
+
+Remove Peers from Downstream Namespace:
+- Remove all BGP neighbors for the selected ASIC namespace via apply-patch.
+- Verify the route table. All routes for BGP neighbors should be gone. Additionally, all kernel and directly connected routes toward the neighbor IPs should be removed.
+- Shutdown local interfaces for the selected ASIC namespace via apply-patch.
+- Verify that the admin status of local interfaces in the selected ASIC namespace is down/down.
+- Perform data traffic tests toward a randomly selected neighbor. Traffic should fail.
+- Perform data traffic tests toward the static route from the randomly selected neighbor. Traffic should fail.
+
+Re-add Peers and Re-enable Interfaces:
+- Change cable lengths.
+- Re-add peers in the downstream namespace. -
+- Re-enable interfaces.
+- Verify that the peers are re-added, BGP sessions are established, and the route table is updated.
+- Verify the buffer profile created for the new cable length in CONFIG_DB, APPL_DB, and ASIC_DB.
+- Perform data traffic tests toward a randomly selected neighbor. Traffic should pass.
+- Perform data traffic tests toward the static route from the randomly selected neighbor. Traffic should pass.
+
+Send PFC Frame Pause/Continue:
+- Send a PFC pause frame.
+- Perform data traffic tests toward the static route. Traffic should not pass.
+- Send a PFC continue frame.
+- Perform data traffic tests toward the static route. Traffic should pass.
```

Comment:
Please add checks for various pkt counters to make sure there are not drops on the interface we newly enabled.